proposals/0273-webengine-projection-mode.md

```diff
@@ -26,34 +26,25 @@ A new template called `WEB_VIEW` should be added. This template should have the
 1. It's supported for in-vehicle WebEngine apps only.
 2. It's available only to the main window. It should not be available for widgets.
 3. It is only available to applications that successfully registered with the new App HMI type (see below section regarding new App HMI Type).
-4. The `WEB_VIEW` template is not allowed to to support textfields, softbuttons, graphics (inside or outside of the webview). DuplicateUpdatesFromWindowID=0 is not allowed for widgets.
+4. The `WEB_VIEW` template must not declare support in its WindowCapabilities for any softButtonCapabilities, the TextFields with name: mainField1-4, mediaClock, mediaTrack, nor the ImageFields with name: softButtonImage, menuIcon, graphic, or secondaryGraphic.

```

Comment:
Please add backticks as shown in https://github.com/smartdevicelink/sdl_evolution/issues/912#issuecomment-637749152.

Reply:
@theresalech Thank you. Done in 96ace2f161629c16b1280f18c852cccb6557758c